proto/cosmos/upgrade/v1beta1/query.proto

```diff
@@ -24,6 +24,7 @@ service Query {
   // stored at the last height of this chain.
   // UpgradedConsensusState RPC not supported with legacy querier
```

Comment:
I would add some comments here that it's deprecated, and a link to the [IBC replacement](https://github.com/cosmos/ibc-go/blob/2c880a22e9f9cc75f62b527ca94aa75ce1106001/proto/ibc/core/client/v1/query.proto#L54)